src/api/todos.js

```diff
@@ -0,0 +1,76 @@
+// Todos table functions 
+import axios from './axios';
+
+// RESTful API endpoint for todo objects
+const endpoint = '/todos';
+
+/**
+ * Get all rows in the todos table.
+ * @returns array of todos 
+ */
+async function getTodos() {
+  const { data } = await axios.get(endpoint);
+  return data;
+}
+
+/**
+ * Insert a new todo in the todos table
+ * @param {*} todo new todo to insert 
+ * @returns true if success, false if not
+ */
+async function postTodo(todo) {
+  const { status } = await axios.post(endpoint, todo);
+  return status === 200;
+}
+
+/**
+ * Update the todo in the todos table. Make sure that id is specified. 
+ * @param {*} todo data to update
+ * @returns true if success, false if not 
+ */
+async function putTodo(todo) {
+  const { status } = axios.put(`${endpoint}/${todo.id}`, todo);
+  return status === 200;
+}
+
+/**
+ * Delete the todo specified by the id parameter.
+ * @param {*} id todo identifier
+ * @returns true if success, false if not 
+ */
+async function deleteTodo(id) {
+  const { status } = await axios.delete(`${endpoint}/${id}`);
+  return status === 200;
+}
+
+/**
+ * Get todo based on the given identifier
+ * @param {*} id todo identifier
+ * @returns todo object
+ */
+async function getTodo(id) {
```

Comment:
Please refer to my previous comment on RESTful HTTP semantics and rest of my notes on the comment. For this API call, please also return the response status code (for error handling)